.github/stale.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an issue or PR is considered stale
+daysUntilStale: 30
+
+# Number of days of inactivity before a stale issue or PR is closed
+daysUntilClose: 30
+
+# Issues or PRs with these labels will never be considered stale
+exemptLabels:
+  - up for grabs
+
+# Label to use when marking an issue or PR as stale
+staleLabel: waiting
+
+# Comment to post when marking an issue or PR as stale
+markComment: >
+  Hi all! This thread has not had any recent activity.
+  Can anyone comment on what's blocking progress?
```

Comment:
I mean I don't see that the new version is _not_ a call to action in any way. Its not direct, that's true. And I would rather not be too direct. The new version seems to have the right tonality and still ask for some work to be done in a friendly way.